package.json

```diff
@@ -23,11 +23,18 @@
   },
   "homepage": "https://github.com/olamilekan000/Politico#readme",
   "dependencies": {
```

Comment:
@olamilekan000 , How you intend to start your app when you don't have nodemon installed?

Reply:
I have nodemon installed globally on my system